src/Tools/README.md

```diff
@@ -4,7 +4,6 @@
 
 The folder contains command-line tools for ASP.NET Core. The following tools are bundled* in the .NET Core CLI. Follow the links below for more details on each tool.
 
-- [dotnet-watch](dotnet-watch/README.md)
```

Comment:
I didn't find dotnet-watch in this directory. So I deleted the link and updated usages of dotnet watch examples below.